### ISSUE TITLE
Curate list of tags to something that is less fine-grained and more intuitive

### ISSUE THREAD

**Issue by grymmy** [hackclub/sprig#1329]
Looking at the global list of tags now, it seems like we have gone too far in making the tags fine-grained as a result of the limited tag-filtering UI.  What we really want is to be able to filter by multiple tags simultaneously, thus allowing you to differentiate between the "puzzle strategy" vs "puzzle adventure" games, for instance.  "Sokoban style", "puzzle-score", and "Sprig Lovers Together" are good examples of uninuitive tag names.

Steps:
1. Brainstorm a list of intuitive/logical tags to a set of acceptable ones.
2. Get approval from @leomcelroy on this new set of tags.
3. Update all games to conform with the new tag list.

Separately, I will be filing a feature request to be able to filter by multiple tags simultaneously.

**Comment by leomcelroy** [comment]
I agree. Let's make a list of 20 tags and then edit the existing tags to use only them.
